src/storage/src/hummock/sstable/forward_sstable_iterator.rs

```diff
@@ -32,6 +32,7 @@ pub trait SstableIteratorType: HummockIterator + 'static {
         sstable: TableHolder,
         sstable_store: SstableStoreRef,
         read_options: Arc<SstableIteratorReadOptions>,
+        read_table_id_range: (StateTableId, StateTableId),
```

Comment:
I think it is more extensible to pass `SstableInfoRef` here instead of the table id range. The implementation of this method can decide what information is needed.

Reply:
done